semantic.cabal

```diff
@@ -305,6 +304,8 @@ library
                      , pretty-show ^>= 1.9.5
                      , profunctors ^>= 5.3
                      , reducers ^>= 3.12.3
+                     , semantic-java ^>= 0
+                     , semantic-python ^>= 0
```

Comment:
Since `Language.Python` now lives in `semantic-python`, we depend on it in `semantic`.